pkl/unmarshal_test.go

```diff
@@ -17,7 +17,10 @@ package pkl_test
 
 import (
 	"bytes"
+	"context"
 	_ "embed"
+	"github.com/apple/pkl-go/pkl/test_fixtures/custom"
+	"github.com/stretchr/testify/require"
```

Comment:
Can you run `goimports -w .` to update the import order here?

If you don't have goimports, you can run `go install golang.org/x/tools/cmd/goimports@latest` to install it.
Details on goimports: https://pkg.go.dev/golang.org/x/tools/cmd/goimports